src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -144,6 +144,15 @@
   <data name="DoNotDisableCertificateValidationMessage" xml:space="preserve">
     <value>The ServerCertificateValidationCallback is set to a function that accepts any server certificate, by always returning true. Ensure that server certificates are validated to verify the identity of the server receiving requests.</value>
   </data>
+  <data name="DoNotDisableHeaderChecking" xml:space="preserve">
+    <value>Do Not Disable Header Checking</value>
```

Comment:
Suggestion: Should this be `Do not disable http header checking`? Also tag @dotpaul for his view

Reply:
Agree, "Do Not Disable HTTP Header Checking" is more specific.